packages/esm-patient-medications-app/src/order-basket/order-basket.component.tsx

```diff
@@ -86,11 +77,14 @@ const OrderBasket: React.FC<OrderBasketProps> = ({ patientUuid, closeWorkspace }
     setIsMedicationOrderFormVisible(true);
   };
 
-  const handleSaveClicked = async () => {
+  const handleSaveClicked = useCallback(async () => {
     const abortController = new AbortController();
 
     setCreatingEncounterError(false);
     let orderEncounterUuid = encounterUuid;
+    // If there's no encounter present, stop the order from being created and create an encounter.
+    // TODO: What? Ok then what are we doing with this orderEncounterUuid returned from createEmptyEncounter?
```

Comment:
IIRC, the idea was to create an encounter and then use the orderEncounterUuid to save the order as part of the encounter. This was really supposed to be a temporary workaround for having a proper encounter context.

Reply:
How do I test this? How do I tell whether there is "an encounter present"?